StudCity.Frontend/src/pages/RecoveryPassword/index.jsx

```diff
@@ -21,49 +26,62 @@ function RecoveryPassword() {
 		setFormState({ ...formState, confirmPassword: event.target.value });
 	}
 
-    return (
-		<div className="w-full h-screen flex relative ">
-					<Svg 
-						type="bgRecoveryPassword" 
-						className="h-full w-full object-cover m-auto " 
-					/>
-			<div className="w-1/2 bg-white  h-[85%] flex flex-col m-auto shadow-form rounded-xl absolute  -translate-x-2/4 -translate-y-2/4 top-1/2 left-1/2">
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+
+		const isOK = await service.recoveryPassword({ id: accountId, password: formState.password });
+
+		if (isOK) {
+			setTimeout(() => navigate("/authenticate"), 2000);
+		}
+		else setTimeout(() => navigate("/forgot-password"), 2000);
```

Comment:
use else after closing curly bracket of if statement

Reply:
Go to ass!